code/index.html

```diff
@@ -2,21 +2,142 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet">
+    <style>
+    a:hover {
+    background-color: grey;
+    }
+    </style>
+    
     <!-- dont forget the meta tag for responsiveness -->
   </head>
-  <body>
+  <body> 
+    
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class= "header"> 
+        <div class="header-text"> 
+          <h1>THE TECH MAGAZINE </h1>
+  
+          <div class="header-img-link" ></div> 
+      
+          <p>FOR CREATIVE PEOPLE</p> 
+
+          <button>learn more</button>
+        </div>
+      </div>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <div class="navbar">
```

Comment:
You could make the class at the navbar instead for in a div. Then you can skip that extra div